pynamodb/models.py

```diff
@@ -897,12 +902,21 @@ def create_table(cls, wait=False, read_capacity_units=None, write_capacity_units
                 if status:
                     data = status.get(TABLE_STATUS)
                     if data == ACTIVE:
-                        return
+                        break
                     else:
                         time.sleep(2)
                 else:
                     raise TableError("No TableStatus returned for table")
 
+        ttl_attribute = cls._ttl_attribute()
+        if ttl_attribute:
+            # Some dynamoDB implementations (eg: dynalite) do not support updating TTLs so
```

Comment:
I'd suggest a kwarg to ignore ttl failures (defaults to `False`) rather than compromising legitimate production usage of TTL because of a local dev server is missing a feature